packages/local_auth/local_auth/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.1.8
+
+* Implement isDeviceSupported for ios.
```

Comment:
There are no changes to the `local_auth` package, so it shouldn't have a changelog or version change.